run-test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+touch test/results.log
```

Comment:
Jó lenne megnézni, hogy létezik-e már, és ha igen, kitörölni. Akkor lehetne egymás után többször is futtatni, pl. lokális teszteléshez.